app/views/layouts/_navbar.html.erb

```diff
@@ -1,9 +1,16 @@
 <%# https://getbootstrap.com/docs/4.5/components/navbar/ %>
 <nav class="navbar navbar-expand-md navbar-dark bg-dark">
   <div class="navbar-brand"><%= link_to I18n.t('index.portal_name'), root_path, class: 'portal-link' %></div>
-  <form class="form-inline">
-    <i class="fa fa-bell" style="color: white; padding: 10px"></i>
-    <a href="<%= search_users_path %>"><i class="fa fa-search" style="color: white; padding: 10px"></i> </a>
-  </form>
+  <% if user_signed_in? %>
+    <form class="form-inline">
+      <% if current_user.contact_requests.any? %>
```

Comment:
Does that mean that the bell is not shown if the user does not have any notifications?

Reply:
No, then there's a white bell without a link (it's in the `else` statement)

Reply:
Oh, sorry :sweat_smile: